CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ To publish a new release run `scripts/release` from the project directory.
 
 ## [Unreleased]
 
+### Changed
+- Use either the default `TERM` enviroment variable or the none default `TERM_PROGRAM` for determining WezTerm terminal (see [GH-185]).
```

Comment:
Please rephrase this to remove the judgemental undertone; `TERM_PROGRAM` is not any less "default" then `TERM`.

Just say that mdcat now checks `TERM` in addition to `TERM_PROGRAM` to identify wezterm.

And please link to this merge request, e.g. GH-186, because that's the PR which actually implements it.

Reply:
According to the comment in wezterm it is not really a standard https://github.com/wez/wezterm/blob/ac5199c21663e39dcce887308a92b5e36b338866/config/src/lib.rs#L1709-L1710

Reply:
Well it speaks of an "emerging de facto standard" which is good enough for me 🙂 More so if it actually works by default 😜

And even if it wasn't we'd be using a nonstandard variable to detect a single nonstandard terminal emulator. I don't really see an issue with that 🙂

Reply:
Yeah, well whatever.